tests/specs/footnote/uat_footnote_popover.spec.js

```diff
@@ -0,0 +1,211 @@
+/**
+ * Checks tags page type
+ */
+
+const { assert } = require('chai');
+
+const Page  = require('../../pageobjects/Page');
+function escape(value) {
+  return value.replace(/([\.:])/g, '\\$1');
+}
+
+describe('footnote popover', function () {
+
+  describe('footnote popover looks and behavior on desktop', function () {
+    this.beforeEach(function() {
+      Page.open('historicaldocuments/frus-history/introduction');
+      return Page.getElement('#touch-detector').then(function () {
+        browser.execute(function() {
+          $('#touch-detector').hide();
+        });
+      });
+    });
+    it('should have a footnote', function () {
+      const footnoteLink = Page.getElement('#content-inner a.note[href^="#fn\\:"][rel=footnote]');
+      assert.equal(footnoteLink.getText(), '1');
+      const footnote = Page.getElement(escape(footnoteLink.getAttribute('href')));
+      assert.equal(footnote.getAttribute('value'), '1');
+      assert.equal(footnote.getAttribute('class'), 'footnote');
+    });
+    it('footnote popover should appear on hover', function () {
+      const footnoteLink = Page.getElement('#content-inner a.note[href^="#fn\\:"][rel=footnote]');
+      assert.equal(Page.getElement('.popover').isDisplayed(), false);
+      footnoteLink.scrollIntoView();
+      footnoteLink.moveTo();
+      Page.waitForVisible('.popover');
+      assert.equal(Page.getElement('.popover').isDisplayed(), true);
+    });
+    it('footnote popover should not have the "back" link', function () {
+      const footnoteLink = Page.getElement('#content-inner a.note[href^="#fn\\:"][rel=footnote]');
+      assert.equal(Page.getElement('.popover').isDisplayed(), false);
+      footnoteLink.scrollIntoView();
+      footnoteLink.moveTo();
+      Page.waitForVisible('.popover');
+      assert.equal(Page.getElement('.popover .footnote-body a.fn-back').isExisting(), true);
+      assert.equal(Page.getElement('.popover .footnote-body a.fn-back').isDisplayed(), false);
+    });
+    it('footnote popover should not disappear with the cursor on it', function () {
+      assert.equal(Page.getElement('.popover').isDisplayed(), false);
+      Page.getElement('#content-inner a.note[href^="#fn\\:"][rel=footnote]').scrollIntoView();
+      Page.getElement('#content-inner a.note[href^="#fn\\:"][rel=footnote]').moveTo();
+      Page.waitForVisible('.popover');
+      assert.equal(Page.getElement('.popover').isDisplayed(), true);
+      Page.getElement('.popover').scrollIntoView();
+      Page.getElement('.popover').moveTo();
```

Comment:
`moveTo()` without arguments is failing currently.
This is the error:
```
from invalid argument: 'y' must be an int
  (Session info: chrome=104.0.5112.79)
```
That means running Chrom / Chromedriver v104 with current test framework at version `@wdio/cli@5.18.6` is probably not supported.